home_task_1/src/components/CreateElement/Counter.js

```diff
@@ -0,0 +1,52 @@
+import { Component, createElement} from 'react';
+
+class CreateElementCounter extends Component {
+  constructor() {
+    super()
+
+    this.state = {
+      counter: 0
+    }
+  }
+
+  increment = () => {
+    this.setState((prevState) => ({ counter: prevState.counter + 1 }));
+  }
+
+  decrement = () => {
+    this.setState((prevState) => ({ counter: prevState.counter - 1 }));
+  }
+
+  render() {
+    return createElement(
+      'div',
+      null,
+        createElement(
+          'h2',
+          { className: 'title' },
+          'Counter'
+        ),
+        createElement(
+          'div',
+          null,
+            createElement('span', { className: 'counter' }, this.state.counter)
+        ),
+        createElement(
+          'div',
+          { className: 'content-btn' },
+          createElement(
+            'button', 
+            { className: 'btn-increment', onClick: () => this.increment() },
+            'Increment'
+          ),
+          createElement(
+            'button',
+            { className: 'btn-decrement', onClick: () => this.decrement()},
+            'Decrement'
+          ),
+      ),
+    )
+  }
+}
+
+export default CreateElementCounter;
```

Comment:
[optional] from my perspective it's better use named exports rather than default ones. To keeps consistence in name all across your apps, since you are able to import default-exported component using any name you want

Reply:
Great recommendation